README.md

```diff
@@ -17,7 +17,8 @@ Read more in the [documentation on ReadTheDocs](http://web3py.readthedocs.io/).
 import json
 import web3
 
-from web3 import Web3, HTTPProvider, TestRPCProvider
```

Comment:
We should probably remove this entire section from the `README` in favor of just using RTD.